.github/workflows/main.yml

```diff
@@ -0,0 +1,53 @@
+# This is a basic workflow to help you get started with Actions
+
+name: CI
+
+# Controls when the workflow will run
+on:
+  # Triggers the workflow on push or pull request events but only for the develop branch
+  push:
+    branches:
+      - main
+      - develop
+      - ch-setup-ci-github-actions-cp-27
```

Comment:
Shouldn't the workflow be triggered only for the develop branch? I don't think it's necessary to include the other branches

Reply:
That line was added for testing the workflow. After the full review. I will send in a clean up commit for full configuration without unnecessary configurations.